views/pulsar/layouts/base.html.twig

```diff
@@ -12,7 +12,7 @@
 <!--[if IE 8]>         <html class="no-js ie8 lt-ie10 lt-ie9"><![endif]-->
 <!--[if IE 9]>         <html class="no-js ie9 lt-ie10"><![endif]-->
 <!--[if gt IE 9]><!--> <html class="no-js"><!--<![endif]-->
-<html class="no-js">
+<html lang="en" class="no-js">
```

Comment:
Correct me if I'm wrong, we don't actually need this line because L:14 will be used for non IE browsers